app/tests/pages/__snapshots__/index.test.tsx.snap

```diff
@@ -188,7 +188,7 @@ exports[`landing It matches the last accepted Snapshot 1`] = `
             <td
               className="jsx-473442302"
             >
-              May 1, 2020
+              Mon Apr 01 2019
```

Comment:
Not sure about this format, can you change `toDateString` to output in the same format as the hardcoded values we had before?